pkg/doc/jose/encrypter_decrypter_test.go

```diff
@@ -556,17 +557,31 @@ func TestJWEEncryptRoundTrip(t *testing.T) {
 							EPK: []byte("somerawbytes"),
 						},
 					},
-					{
+				}
+
+				if tc.nbRec > 1 {
+					badJWE.Recipients = append(badJWE.Recipients, &ariesjose.Recipient{
 						EncryptedKey: "someOtherKey",
 						Header: &ariesjose.RecipientHeaders{
 							EPK: []byte("someotherrawbytes"),
 						},
-					},
+					})
 				}
 
 				_, err = jweDecrypter.Decrypt(badJWE)
-				require.EqualError(t, err, "jwedecrypt: failed to build recipients WK: unable to read "+
-					"JWK: invalid character 's' looking for beginning of value")
+				if tc.nbRec == 1 {
+					errStr := "jwedecrypt: failed to unwrap cek: [unwrapKey: deriveKEKAndUnwrap:" +
```

Comment:
couldn't errStr be a field in the testcase, instead of being constructed through some logic based on the testcase parameters?

Reply:
although this is local error for single recipient messages, it's a sub-test, but I guess I can add test case error string for this specific case..

Reply:
updated